packages/react/src/Pagination/Pagination.tsx

```diff
@@ -124,30 +124,21 @@ export const Pagination = forwardRef(
     }
 
     return (
-      <nav {...restProps} className={clsx('ams-pagination', className)} ref={ref} aria-label="Paginering">
+      <nav {...restProps} className={clsx('ams-pagination', className)} ref={ref}>
```

Comment:
You would still be able to supply an aria-labbeledby attribute, which is ok? https://web.dev/articles/website-navigation#add_a_landmark

Reply:
People can still do that via the `{...restProps}`, can't they?